### PR TITLE
Remove vault.centos.org repo for CentOS as it's EOL and support CentOS Stream 8

### PR DESCRIPTION
**Pull request by rayandas**

### CODE REVIEW

File: dist/24.0.sh
Comment (snasovich) [inline dist/24.0.sh:1]: GH shows that this file got deleted on PR. Is this a glitch and the file is still there? I vaguely recall it having issues with symlink files.

File: pkg/24.0.0/24.0.0.diff [hunk 1]
```diff
@@ -94,29 +94,20 @@
  		;;
  
  		*)
-@@ -553,14 +578,17 @@
- 					set -x
- 				fi
- 				$sh_c "DEBIAN_FRONTEND=noninteractive apt-get install -y -qq $pkgs >/dev/null"
-+				start_docker
```
Comment (snasovich) [inline pkg/24.0.0/24.0.0.diff:101]: Why are we removing addition of `start_docker` - doesn't seem to match description of what the PR does.

File: pkg/24.0.0/config.sh [hunk 2]
```diff
@@ -5,6 +5,6 @@ DOCKER_GIT_COMMIT="c2de0811708b6d9015ed1a2c80f02c9b70c8ce7b"
 TEST_OS_IMAGE_NAME=(redhat/ubi9 oraclelinux centos ubuntu rockylinux)
 TEST_OS_IMAGE_TAG[0]="9.0.0"
 TEST_OS_IMAGE_TAG[1]="7 8"
-TEST_OS_IMAGE_TAG[2]="centos7 centos8"
+TEST_OS_IMAGE_TAG[2]="centos7"
```
Comment (snasovich) [inline pkg/24.0.0/config.sh:8]: Can we have CentOS 8 Stream added to the automated testing?

Comment (rayandas) [reply pkg/24.0.0/config.sh:8]: Added Centos Stream8. The images are present only on quay.io

File: pkg/24.0.0/24.0.0.diff (hunk 1 above)
Comment (rayandas) [reply pkg/24.0.0/24.0.0.diff]: Ah! that was a mistake. Updated it.

File: dist/24.0.sh
Comment (rayandas) [reply dist/24.0.sh:1]: Fixed

**Review by snasovich — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**